include/HDF5_Typedef.h

```diff
@@ -74,6 +74,7 @@ struct KeyInfo_t
    double dTime_AllLv[NLEVEL];
 #  ifdef GRAVITY
    double AveDens_Init;             // AveDensity_Init
+   double GREP_Center[3];           // GREP center
```

Comment:
Also store this variable in the `InputPara_t` structure together with other GREP variables. I know this is a bit confusing and redundant. We may consider removing `KeyInfo_t` completely in the future.

Update `Init_ByRestart_HDF5.cpp` and `Output_DumpData_Total_HDF5.cpp` accordingly.

Reply:
@Inhyeok-Song Upon reflection, I withdraw the above suggestion since `GREP_Center[]` is not an input parameter. Besides, your current implementation is consistent with other existing key variables like `double Time[NLEVEL]`.